app/views/blackouts/new_recurring.html.erb

```diff
@@ -1,16 +1,11 @@
 <% title "New Recurring Blackout" %>
 
-<div id="form_recurring">
-  <%= simple_form_for(@blackout, url: {controller: 'blackouts', action: 'create_recurring' }) do |f| %>
-    <%= f.error_notification %>
-
-    <div class="form-inputs">
-      <%= hidden_field_tag "recurring", true %>
-      <%= f.input :days, collection: days_of_the_week_short_with_index,
-        value_method: :first, label_method: :last, as: :check_boxes,
-        item_wrapper_class: 'inline', required: true %>
-
-      <%= render partial: 'form', locals: {f: f} %>
-
-  <% end %>
-</div>
+<%= simple_form_for @blackout, url: {controller: 'blackouts', action: 'create_recurring' },
+html: { class: 'form-horizontal' }, wrapper: :horizontal_form do |f| %>
+  <%= f.error_notification %>
+  <%= hidden_field_tag "recurring", true %>
+  <%= f.input :days, collection: days_of_the_week_short_with_index,
+    value_method: :first, label_method: :last, as: :check_boxes,
+    item_wrapper_class: 'inline', required: true %>
```

Comment:
Yea, when I was trying to decipher the bootstrap initializer I had a really hard time finding documentation. One thing to note - you added the `inline` class to the checkbox `span`s but didn't actually specify that the class should have the `display: inline-block` attribute. Adding the following somewhere should fix this:

```
.checkbox.inline {
  display: inline-block;
  padding-right: 15px;
}
```